### PR TITLE
Added Ruby 2.4.1 to Travis configuration

### PR DESCRIPTION
**Pull request by portertech**
Related to https://github.com/sensu-plugins/sensu-plugins-feature-requests/issues/27

### CODE REVIEW

**Comment by portertech** [comment]
@majormoses Sensu 1.0.0 will be using Ruby 2.4.1, so I this version as well.

**Review by majormoses — CHANGES_REQUESTED** [review]
please add that to the deploy section aws well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ### Changed
 - Removed Ruby 1.9 support
 - Updated sensu-plugin dependency from `= 1.2.0` to `~> 1.3`
+- Updated Travis configuration to include Ruby 2.4.1
```
Comment (majormoses) [inline CHANGELOG.md]: move this up to be under `## Unreleased` under a `## Added` headers.

Comment (majormoses) [reply CHANGELOG.md]: no changing existing versions...hehe...

**Review by majormoses — CHANGES_REQUESTED** [review]

**Review by majormoses — APPROVED** [review]
LGTM

**Review by majormoses — APPROVED** [review]
LGTM

**Pull request merged**